portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,122 @@
+            <div onclick="addRandomFact()" class="buttontext">
+                <span>C</span>
+                <span>L</span>
+                <span>I</span>
+                <span>C</span>
+                <span>K</span>
+                <span> </span>
+                <span>H</span>
+                <span>E</span>
+                <span>R</span>
+                <span>E</span>
+            </div>
+            <div id="funfact-container"></div>
+
+            <div class = "oneliner">
+                Curious to learn more? Check out the additional content linked below!
+            </div>
+        </div>
+
+        <table>
+            <tbody>
+                <td> <div class="paragraph"> 
+                    I've been selected to partipate in fellowships and other programs with companies like D.E. Shaw, Google, Microsoft, and the National Society of Black Engineers (NSBE). A more thorough summary of my professional experience, education, and skills can be viewed on my LinkedIn profile. Feel free to connect! </td>
+                    </div>
+                <td>
+                    <a href="https://www.linkedin.com/in/ashajohnson/" >
+                        <div class="buttontext">
+                            <span>L</span>
+                            <span>I</span>
+                            <span>N</span>
+                            <span>K</span>
+                            <span>E</span>
+                            <span>D</span>
+                            <span>I</span>
+                            <span>N</span>
+                        </div>
+                    </a>
+                </td>
+            </tbody>
+
+             <tbody>
+                <td> <div class="paragraph"> 
+                    I have experience working with C++, CSS, HTML, Unix/Linux, and Python. A few of my coding projects can be viewed on my GitHub profile. </td>
+                    </div>
+                <td>
+                    <a href="https://github.com/AshaBrianna" >
+                        <div class="buttontext">
+                            <span>G</span>
+                            <span>I</span>
+                            <span>T</span>
+                            <span>H</span>
+                            <span>U</span>
+                            <span>B</span>
+                        </div>
+                    </a>
+                </td>
+            </tbody>
+     
+            <tbody>
+                <td> <div class="paragraph"> 
+                    While I also enjoy back-end design, I love the artistic aspects of front-end design and UX Engineering. In addition to graphic and product design, I also love expressing myself artistically through mediums like painting, drawing, watercolors, and ink prints. While most of my work is for fun, I also take commissions. A portfolio-style collection of some of my artwork can be viewed on by art website (Note: The site initially loads extremely slowly). </td>
```

Comment:
Small comment on style: it can be helpful to limit lines to 80 characters to make the code more readable